src/main/resources/anniv6Resources/localization/eng/TheFog/Eventstrings.json

```
@@ -0,0 +1,17 @@
+{
+  "${ModID}:StandingStones": {
+    "NAME": "Standing Stones",
+    "DESCRIPTIONS": [
+      "An #podd #pformation #pof #pstones appears from within the fog. Whilst standing in the centre of the circle they form, you can see the #bnight #bsky above - the ~real~ night sky, from before the #rSpireblight. NL NL Though you don't understand this place in the slightest, you feel #gsafe here. Why not stay for a while to recuperate?",
```

Just switching this to American English for consistency.
```suggestion
      "An #podd #pformation #pof #pstones appears from within the fog. Whilst standing in the center of the circle they form, you can see the #bnight #bsky above - the ~real~ night sky, from before the #rSpireblight. NL NL Though you don't understand this place in the slightest, you feel #gsafe here. Why not stay for a while to recuperate?",
```